File: features/content/feed/src/main/java/com/tokopedia/feedplus/data/pojo/FeedMeta.java
package com.tokopedia.feedplus.data.pojo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * @author by milhamj on 17/07/18.
 */

public class FeedMeta {
    @SerializedName("has_next_page")
    @Expose
    private boolean hasNextPage;

    public void setHasNextPage(boolean hasNextPage) {
        this.hasNextPage = hasNextPage;
    }

    public boolean isHasNextPage() {
        return hasNextPage;
    }
}

File: features/content/feed/src/main/java/com/tokopedia/feedplus/data/pojo/FeedMeta.kt
package com.tokopedia.feedplus.data.pojo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * @author by astidhiyaa on 29/08/22
 */
data class FeedMeta(
    @SerializedName("has_next_page")
    @Expose val hasNextPage: Boolean = false
)
